TeamMemberFiles/Christian.Rodriguez.txt

```diff
@@ -0,0 +1,5 @@
+Christian Rodriguez
+You're more than welcome to call me Christian or Chris I really dont mind.
+I am in section 294
+At the moment I am best at MySql and Java, so I would say I probably prefer those.
+I am about to get married in exactly 3 weeks, October 13, and will be traveling out of the country the last week of this course.
```

Comment:
Congratulations on the wedding :).  By the last week of the course you'll be all done here.

Please make sure to clean this up on your personal repo to avoid exposing any private information to the public.